sql/core/src/main/scala/org/apache/spark/status/api/v1/sql/SqlResource.scala

```diff
@@ -94,7 +94,8 @@ private[v1] class SqlResource extends BaseAppResource {
 
     val duration = exec.completionTime.getOrElse(new Date()).getTime - exec.submissionTime
     val planDetails = if (planDescription) exec.physicalPlanDescription else ""
-    val nodes = if (details) printableMetrics(graph.allNodes, exec.metricValues) else Seq.empty
+    val nodes = if (details) printableMetrics(graph.allNodes,
+      Option(exec.metricValues).getOrElse(Map.empty)) else Seq.empty
```

Comment:
I think the format is:
```scala
    val nodes = if (details) {
      printableMetrics(graph.allNodes, Option(exec.metricValues).getOrElse(Map.empty))
    } else {
      Seq.empty
    }
```
Please see https://github.com/databricks/scala-style-guide#curly-braces:
<img width="872" alt="image" src="https://user-images.githubusercontent.com/5399861/159457818-d4db7df6-aa4a-4d05-a80f-1f918ea2c946.png">